pkgs/development/interpreters/python/python2/mk-python-derivation.nix

```diff
@@ -26,6 +26,23 @@
 , eggInstallHook
 }:
 
+let
```

Comment:
I am not sure about this but we could we just not add support for python 2 to move it closer to removal?

Reply:
It's still relevant because it is used by `pypy2.pkgs.buildPython*`. and that PyPy says that they will continue to support PyPy 2, as RPython (the language they use to implement the PyPy interpreter) is essentially a Python 2 subset.